common-jvm/src/main/java/org/ergoplatform/utils/TimeUtils.kt

```diff
@@ -7,16 +7,18 @@ fun getTimeSpanString(
     stringProvider: StringProvider
 ): String {
     val timeSpanString: String
-    if (seconds < 60) {
+    if (seconds < 30) {
```

Comment:
Not sure. The method has a lot of magic numbers, but on second glance the numbers are quite obvious. It could get harder to read when the numbers are moved to the file header?

Reply:
What is not clear is that is there any connection between them. Also, having a constant you can also put a description about  semantics. And the good name should also reflect the semantics.